scripts/dasharo-deploy

```diff
@@ -757,7 +1041,7 @@ update() {
     echo "Rebooting"
     sleep 1
     send_dts_logs
```

Comment:
Not your change but in case of updating EC logs won't be sent

Reply:
Reworked here 5052984bf86d97496d9ab18eee386e1b8bec5104.